client/views/helpers/qr_code/qr_code.scss

```diff
@@ -0,0 +1,35 @@
+.modaltopmessage > h2 {
+  line-height: 25px;
+  margin-top: 5px;
+  color: black;
+}
+
+.qr_codeouter {
+  padding: 10px;
+  display: none;
+  background-color: #FFF;
+  border: 2px solid #e9edf0;
+  border-radius: 5px;
+  display: block;
+  width: auto;
+  height: auto;
+  margin-top: 120px;
+  margin-bottom: 120px;
+  z-index: 9999999999999999;
```

Comment:
@singhsanket143 it's not a valid value https://stackoverflow.com/questions/491052/minimum-and-maximum-value-of-z-index/25461690#25461690

Reply:
@peter-hank I have updated the z-index corresponding to chrome browser